app/services/moves/finder.rb

```diff
@@ -4,7 +4,7 @@ module Moves
   class Finder
     attr_reader :filter_params, :ability
 
-    MOVE_INCLUDES = [:from_location, :to_location, profile: { person_escort_record: [:framework, :framework_responses, framework_flags: :framework_question] }, person: %i[gender ethnicity]].freeze
+    MOVE_INCLUDES = [:court_hearings, :prison_transfer_reason, :original_move, profile: [documents: { file_attachment: :blob }, person_escort_record: [:framework, :framework_responses, framework_flags: :framework_question]], person: %i[gender ethnicity profiles], from_location: %i[locations_suppliers suppliers], to_location: %i[locations_suppliers suppliers], allocation: %i[to_location from_location]].freeze
```

Comment:
do we need to update some tests to reflect these changes?

Reply:
I don't think we do, as this is just eager loading the records to avoid N+1 and slower queries. I was able to obtain this list from the [Bullet gem](https://github.com/flyerhzm/bullet) which highlights where a query can be optimised